pkg/pex/option.go

```diff
@@ -75,24 +83,6 @@ func WithBootstrapPeers(peers ...peer.AddrInfo) Option {
 	}
 }
```

Comment:
>The supplied instance will be called with 'opt' whenever
>the PeeerExchange is unable to connect to peers in its cache.

I think this should be 'peers' instead of opt, no?  I also think this would be clearer:

>Namespaces will be bootstrapped using the supplied peers whenever the PeerExchange is unable to connect to peers in its cache.

(P.S.:  there's also a typo in `PeeerExchange`)

Reply:
I went ahead and fixed this as well.